coral/src/app/features/approvals/components/ApprovalsLayout.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box } from "@aivenio/aquarium";
+import { Pagination } from "src/app/components/Pagination";
+import { ReactElement } from "react";
+
+type ApprovalsWrapperProps = {
+  filters: ReactElement[];
+  // add <typeof TypeApprovalsComponentTable> later
```

Comment:
Can it be `ReactElement<typeof DataTable>` ?

Reply:
Yes it could for now!

Reply:
Not forever? I think we only want to use `DataTable` for approvals, right?

Reply:
If we do eg. `<typeof SchemaApprovalsRequest>` (+ others) the typing will be more strict and precise, I think that would be nice!